addons/sys_core/fnc_remoteStartSpeaking.sqf

```diff
@@ -102,6 +102,9 @@ private _result = false;
                 _unit setVariable [QGVAR(currentSpeakingRadio), _radioId];
                 private _speakerRadio = [];
                 private _nearRadios = [ACRE_LISTENER_POS, 150] call EFUNC(sys_radio,nearRadios);
+                if (!isNull(findDisplay 312)) then { //Zeus
```

Comment:
i am not sure but is this faster or just `isNull curatorCamera`?

Reply:
This is what I use in my mod, I am pretty sure I ran tests, but I'll double check